app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts

```diff
@@ -338,7 +339,7 @@ const ShelleyLedgerCryptoProvider = async ({
     const poolOwners: LedgerTypes.PoolOwner[] = stakepoolOwners.map((owner) => {
       // TODO: helper function for slicing first bit from staking address so its stakingKeyHash
       return !Buffer.compare(
```

Comment:
Overriding the value to an empty string seems really fragile, I'd rather do something like this

```suggestion
      return owner.stakingKeyHashHex && !Buffer.compare(
```